bravado/mapping/exception.py

```diff
@@ -0,0 +1,9 @@
+class SwaggerError(Exception):
+    """Raised when an error is encountered."""
+
+    def __init__(self, msg, cause=None):
+        """
+        :param msg: String message for the error.
+        :param cause: Optional exception that caused this one.
+        """
+        super(Exception, self).__init__(msg, cause)
```

Comment:
This should either go into `bravado/exception.py` or be renamed to be explicitly for mapping exceptions.

I think we already have this exact class in `bravado/exception.py`